client/src/pages/Scheduler/Scheduler.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { Box } from '@material-ui/core';
+import useStyles from './useStyles';
+import { Typography } from '@material-ui/core';
+import { useHistory } from 'react-router-dom';
+import Calendar from 'react-calendar';
+import moment from 'moment-timezone';
+import ScheduleIcon from '@material-ui/icons/Schedule';
+import FormControl from '@material-ui/core/FormControl';
+import BuildTimeZones from './BuildTimeZones';
+import TimePopulator from './TimePopulator';
+
+interface disableDateProp {
+  activeStartDate: Date;
+  date: Date;
+  view: string;
+}
+
+export default function Scheduler(): JSX.Element {
+  const history = useHistory();
+  const classes = useStyles();
+
+  const username = 'TEMP USER';
+  const duration = 60;
+  const availableDays = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'];
```

Comment:
Just to clarify, initially on scheduler we are assuming only available Monday to Friday but not Saturday or Sunday?

Reply:
The constants are actually there as a placeholder for testing purposes. 

I was planning to get information of the user's availablities from the db and replace them.
I should have left the comments about these. This is my bad.